### DISCUSSION TITLE
Function to insert pictures does not work

### DISCUSSION THREAD

**Discussion by matthias-ronge** [kitodo/kitodo-production#4404] [Q&A] [answered]
There is a function in the pagination dialog to insert images:

![Screenshot of function to insert images](https://user-images.githubusercontent.com/3040657/105078305-13dc6980-5a8e-11eb-952f-6077897eb0a8.png)

If you enter a number here and press insert, the metadata editor crashes.

![Screenshot with stack trace](https://user-images.githubusercontent.com/3040657/105078476-5b62f580-5a8e-11eb-9541-2d26c4bd8cbf.png)

Another problem: The functionality is not self-explanatory. What should happen here, and what does that have to do with pagination? Changers from Production 2.x expect a file selection dialog to open here so that the user can upload a missing image (TIFF):

![Screenshot from Production 2](https://user-images.githubusercontent.com/3040657/105079294-80a43380-5a8f-11eb-9cdd-0653b677d4d1.PNG)

**Goal:** The functionality has to be clarified. If it makes sense, it needs an understandable narrative and, of course, needs to be fixed. Otherwise the input function of the dialog has to be removed.

**Comment by solth** [comment, marked as answer]
We _do_ have the option to upload images in the metadata editor via the context menu now:
![Bildschirmfoto 2021-10-08 um 10 25 38](https://user-images.githubusercontent.com/19183925/136523760-8d64336d-dc5a-416b-a6ec-e5e034c29d43.png)
Does that cover the required functionality?

**Comment by IkramMaalej** [comment]
I think that the input function of the Pagination dialog has to be removed.

**Comment by matthias-ronge** [comment]
I would say you are both right: The functionality is that hidden in the pop-up menu (I hadn't looked for it at that point) **and** the specified buttons in the pagination dialog should be removed.
